### ISSUE TITLE
ability to customise the request

### ISSUE THREAD

**Issue by yschimke** [launchdarkly/okhttp-eventsource#28]
I use the request tag in okhttp, but can't easily configure it using the current api.  I ended up with the following hack

```
package com.launchdarkly.eventsource

import com.baulsupp.oksocial.credentials.DefaultToken
import okhttp3.Request

class TweakedEventSource(builder: Builder): EventSource(builder) {
  override fun buildRequest(): Request {
    return super.buildRequest().newBuilder().tag(DefaultToken).build()
  }
}
```

**Comment by eli-darkly** [comment]
Hi. I'm sorry we overlooked this request for so long, but this is now implemented in version 1.9.0.

**Comment by yschimke** [comment]
No need to apologise.  Thanks for adding.
